dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/nonfinancials/EuTaxonomyActivity.kt

```diff
@@ -29,9 +26,17 @@ data class EuTaxonomyAlignedActivity(
     val activityName: Activity,
     val naceCodes: List<String>?,
     val share: RelativeAndAbsoluteFinancialShare?,
-    @field:Schema(example = JsonExampleFormattingConstants.SUBSTANTIAL_CONTRIBUTION_CRITIREA)
-    val substantialContributionCriteria: Map<EnvironmentalObjective, BigDecimal>?,
-    @field:Schema(example = JsonExampleFormattingConstants.DNSH_CRITIREA)
-    val dnshCriteria: Map<EnvironmentalObjective, YesNo>?,
+    val substantialContributionToClimateChangeMitigation: BigDecimal?,
+    val substantialContributionToClimateChangeAdaption: BigDecimal?,
+    val substantialContributionToSustainableUseAndProtectionOfWaterAndMarineResources: BigDecimal?,
+    val substantialContributionToTransitionToACircularEconomy: BigDecimal?,
+    val substantialContributionToPollutionPreventionAndControl: BigDecimal?,
+    val substantialContributionToProtectionAndRestorationOfBiodiversityAndEcosystems: BigDecimal?,
+    val dnshToClimateChangeMitigation: YesNo?,
```

Comment:
Does it make sense that someone can select `No` for any of the DNSH criteria. Isn't that a requirement for something to be considered Taxonomy aligned?

Reply:
Will bring this up with Julia.

Reply:
This is part of a separate discussion/story now